### ISSUE TITLE
Tabular Data

### ISSUE THREAD

**Issue by jjaviergalvez** [diffgram/diffgram#1332]
Exploring the library, I see a lack of support for tabular data labeling. It would be great to have that feature.

**Comment by vitalii-bulyzhyn** [comment]
@jjaviergalvez you are right, we do not support tabular at the moment. It's been in our horizon for a while, but we are decided to concentrate on other interfaces first

**Comment by PJEstrada** [comment]
Hi @jjaviergalvez,

Do you have some specific features, UI ideas layout or use cases you might want to share so we can analyze them?

**Comment by jjaviergalvez** [comment]
Thank you. 

@PJEstrada As an example, it could be product matching by attributes-titles-descriptions.

**Comment by PJEstrada** [comment]
Thanks.  Do you have any reference interfaces,  UI or projects  to understand what you mean?

**Comment by jjaviergalvez** [comment]
Have a look at this [post](https://hackernoon.com/product-matching-with-ml-transforms-the-retail-industry). As a layout, I imagine something similar to this 
<img width="596" alt="image" src="https://user-images.githubusercontent.com/24707368/222859924-f820611c-1918-4d74-9cff-bf920ab207f0.png">

**Comment by PJEstrada** [comment]
Thanks @jjaviergalvez,

One thing you can already do is use our current version that could potenitally support this is use our 
Text Annotation tool: https://diffgram.readme.io/docs/tutorial-overview

And attach Global Attributes: https://diffgram.readme.io/docs/apply-global-attributes-1

To the text
